src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -307,21 +307,16 @@ export default class ChangeBusinessType extends Mixins(CommonMixin, SharedMixin)
   /** Reset company type values to original. */
   private resetType () {
     this.setEntityType(this.getEntitySnapshot?.businessInfo?.legalType)
-    this.emitHaveChanges(false)
```

Comment:
The components no longer emit whether they have changes, and these values are no longer stored. This was used by BEN IA corrections only to determine if there was a change.

Now, BEN IA corrections use getters that compare original vs new to determine if there was a change (same as the other filings in this UI).